src/components/DataTable/DataTable.scss

```diff
@@ -132,18 +132,7 @@ $breakpoint: 768px;
 }
 
 .Cell-sortable {
-  cursor: pointer;
-
-  &:hover {
-    .Heading {
-      @include recolor-icon(color('indigo'));
-      color: color('indigo');
-    }
-
-    .Icon {
-      opacity: 1;
```

Comment:
Any idea what the opacity here was for? Any issues with removing it?

Reply:
That's the hover style that starts at 0.